packages/vue-app/template/store.js

```diff
@@ -8,74 +8,28 @@ let storeData = {}
 let files;
 
 void function updateModules() {
-  files = require.context('@/<%= dir.store %>', true, /^\.\/(?!<%= ignorePrefix %>)[^.]+\.(<%= extensions %>)$/)
-  const filenames = files.keys()
-
-  // Check if {dir.store}/index.js exists
-  const indexFilename = filenames.find(filename => filename.includes('./index.'))
-
-  if (indexFilename) {
-    storeData = getModule(indexFilename)
-  }
+  <% stores.some(s => {
+    if(s.src.indexOf('index.') === 0) { %>
+  storeData = require('@/<%= dir.store %>/<%= s.src %>')
+  <% return true }}) %>
 
   // If store is not an exported method = modules store
   if (typeof storeData !== 'function') {
     // Store modules
     if (!storeData.modules) {
       storeData.modules = {}
     }
+    <% stores.forEach(s => {
```

Comment:
Why not moving this part to builder/generateRoutes for making templates cleaner?

Reply:
Builder:L394

Reply:
It should, but I didn't touch the existing logic in template, I'll try.